### PR TITLE
#14 Edit complex task

### PR DESCRIPTION
**Pull request by maciejhein**
Screenshot 2019-10-25 at 13 22 46

mockup

Screenshot 2019-10-24 at 09 58 17

### CODE REVIEW

File: src/screens/planItemActivity/ComplexTaskItem.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    </View>
+  </Card>
+);
+
+const styles = StyleSheet.create({
+  card: {
+    flexDirection: 'row',
+    width: '100%',
+    paddingVertical: 0,
+    paddingHorizontal: 0,
```
Comment (kwbak) [inline src/screens/planItemActivity/ComplexTaskItem.tsx]: Maybe just `padding: 0` would work?

File: src/screens/planItemActivity/ComplexTaskItem.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  name: string;
+  image: string;
+  time: string;
+}
+
+export const ComplexTaskItem: SFC<Props> = ({ name, image, time }) => (
+  <Card style={styles.card}>
+    <View style={styles.leftContainer}>
+      <IconButton name="eye-outline" size={24} color={palette.primary} />
+      <IconButton name="delete" size={24} color={palette.textInputPlaceholder} />
```
Comment (kwbak) [inline src/screens/planItemActivity/ComplexTaskItem.tsx]: Missing icon type: `type="material"` to keep all Icon in the same style

File: src/screens/planItemActivity/ComplexTaskItem.tsx (hunk 1 above)
Comment (mgolawski) [reply src/screens/planItemActivity/ComplexTaskItem.tsx]: `padding: 0`

File: src/screens/planItemActivity/PlanItemForm.tsx [hunk 3]
```diff
@@ -90,13 +90,13 @@ export class PlanItemForm extends React.PureComponent<Props, State> {
...
         </View>
-        <Card style={styles.card}>
-          {this.state.taskType === PlanItemType.SimpleTask ? (
-            <PlanItemSimpleTask planItem={this.props.planItem} formikProps={formikProps} />
-          ) : (
-            <PlanItemComplexTask planItem={this.props.planItem} />
-          )}
-        </Card>
+        {this.state.taskType === PlanItemType.SimpleTask ? (
+          <SafeAreaView style={styles.simpleTaskContainer}>
```
Comment (mgolawski) [inline src/screens/planItemActivity/PlanItemForm.tsx]: This `SafeAreaView` should be inside `SimpleTask` component

**Pull request merged**